src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,38 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.controllers;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+public class TelevisionsController {
+    @GetMapping("/televisions")
+    @ResponseBody
+    public String getAllTelevisions() {
+        return "List Televisions";
+    }
+
+    @GetMapping("/televisions/{model}")
+    @ResponseBody
+    public String getTelevision(@PathVariable String model) {
+        return "Type television" + model;
+    }
+
+    @PostMapping("/api/television")
+    public <TelevisionRequest> ResponseEntity<String> createTelevision(@RequestBody TelevisionRequest televisionRequest) {
+        String responseMessage = String.format("Television received: %s", televisionRequest.getClass());
```

Comment:
De `.getClass()` methode returned in dit geval "TelevisionRequest", maar je weet al dat het die klasse is. 
Ik kan me voor stellen dat er in de TelevisionRequest klassen een "name" staat met een getter en een setter, dus dan zou ".getName()" een meer logische oplossing zijn. In Postman kun je dat dan in JSON invullen in de request body als: 

{ 
"name": "samsung"
}

Hier gaan we het deze week o.a. ook over hebben in de les